Packs/Slack/Scripts/SlackAskV2/SlackAskV2.py

```diff
@@ -0,0 +1,137 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import dateparser
+
+STYLES_DICT = {
+    'black': '',
+    'green': 'primary',
+    'red': 'danger'
+}
+
+DATE_FORMAT = '%Y-%m-%d %H:%M:%S'
+
+
+def create_blocks(text: str, entitlement: str, options: list, reply: str) -> list:
+    value = json.dumps({
+        'entitlement': entitlement,
+        'reply': reply
+    })
+    blocks: list = [{
+        'type': 'section',
+        'text': {
+            'type': 'mrkdwn',
+            'text': text
+        }
+    }]
+
+    elements = []
+    for option in options:
+        element = {
+            'type': 'button',
+            'text': {
+                'type': 'plain_text',
+                'emoji': True,
+                'text': option['text']
+            },
+            'value': value
+        }
+        if 'style' in option:
+            element['style'] = option['style']
+        elements.append(element)
+    if elements:
+        actions = {
+            'type': 'actions',
+            'elements': elements
+        }
+        blocks.append(actions)
+
+    return blocks
+
+
+def main():
+    res = demisto.executeCommand('addEntitlement', {'persistent': demisto.get(demisto.args(), 'persistent'),
+                                                    'replyEntriesTag': demisto.get(demisto.args(), 'replyEntriesTag')})
+    if isError(res[0]):
+        demisto.results(res)
+        sys.exit(0)
+    entitlement = demisto.get(res[0], 'Contents')
+    option1 = demisto.get(demisto.args(), 'option1')
+    option2 = demisto.get(demisto.args(), 'option2')
+    extra_options = argToList(demisto.args().get('additionalOptions', ''))
+    reply = demisto.get(demisto.args(), 'reply')
+    response_type = demisto.get(demisto.args(), 'responseType')
+    lifetime = demisto.get(demisto.args(), 'lifetime')
+    slack_instance = demisto.get(demisto.args(), 'slackInstance')
+    try:
+        expiry = datetime.strftime(dateparser.parse('in ' + lifetime, settings={'TIMEZONE': 'UTC'}),
+                                   DATE_FORMAT)
+    except Exception:
+        expiry = datetime.strftime(dateparser.parse('in 1 day', settings={'TIMEZONE': 'UTC'}),
+                                   DATE_FORMAT)
+    default_response = demisto.get(demisto.args(), 'defaultResponse')
+
+    entitlement_string = entitlement + '@' + demisto.investigation()['id']
+    if demisto.get(demisto.args(), 'task'):
+        entitlement_string += '|' + demisto.get(demisto.args(), 'task')
+
+    args = {
+        'ignoreAddURL': 'true',
+        'using-brand': 'SlackV3'
```

Comment:
@itay4 @amshamah419 do we want the `using-brand`?

Reply:
I think since we are using `send-notification` we should just to prevent other integrations from firing. I'm open to other ideas though since I don't like the idea of bumping the script for any slack version change.